docs/source/basics/latch_develop_example.md

```diff
@@ -0,0 +1,251 @@
+```
+$ latch develop .
+```
+* Now, re-run the test script: 
+```
+>>> run-script scripts/main.py
+```
+Your script should now run successfully!
+
+**Where are my outputs?**
+* Let's inspect the return statement of the `_build_index` task inside `wf/__init__.py`:
+```python
+@small_task
+def _build_index(ref_genome: LatchFile = LatchFile("latch:///wgs/ref_genome/ecoli_rel606.fasta")) -> LatchDir: 
+    _bwa_cmd = [
+        "bwa", 
+        "index",
+        ref_genome.local_path
+    ]
+    subprocess.run(_bwa_cmd)
+    output = os.path.dirname(os.path.abspath(ref_genome.local_path))
+
+    return LatchDir(output, "latch:///wgs/ref_genome")
+```
+* We can see the task is returning a `LatchDir` with the filepath `latch:///wgs/ref_genome`, which indicates that the output files are located inside the `/wgs/ref_genome` folder on Latch!
+
+![Outputs](../assets/latch-develop-example/ref_genome.png)
+
+## Debugging Subsequent Tasks
+* Similarly to how we tested our first task, we can also call the second task from `scripts/main.py` like so: 
+```python
+from wf import _build_index, align_reads, convert_to_bam, sort_bam, variant_calling
+from latch.types import LatchFile, LatchDir
+
+ref_genome_dir = _build_index(ref_genome = LatchFile("latch:///wgs/ref_genome/ecoli_rel606.fasta")) 
+
+align_reads(ref_genome_dir = ref_genome_dir, read1 = LatchFile("latch:///wgs/trimmed_fastqs/SRR2584863_1.trim.sub.fastq"), read2 = LatchFile("latch:///wgs/trimmed_fastqs/SRR2584863_2.trim.sub.fastq"))
+```
+* Here, we are passing the output of the first task, `latch:///wgs/ref_genome`, as the input to the second task. 
+* Run the test script: 
+```
+>>> run-script scripts/main.py
+
+Syncing your local changes... 
+Could not find /Users/hannahle/Documents/GitHub/simple-variant-calling/buggy-wf/data - skipping
+Finished syncing. Beginning execution and streaming logs:
+Finished downloading ecoli_rel606.fasta.amb
+Finished downloading ecoli_rel606.fasta.ann
+Finished downloading ecoli_rel606.fasta.pac
+Finished downloading ecoli_rel606.fasta.sa
+Finished downloading ecoli_rel606.fasta
+Finished downloading ecoli_rel606.fasta.bwt
+['/tmp/flyte-4jp7wtbm/sandbox/local_flytekit/638a02102804a22fa669f18a715b580a/ecoli_rel606.fasta']
+====================
+2022-11-17 00:42:29,046 flytekit ERROR Exception occured when executing task: Failed to get data from latch:///wgs/trimmed_fastqs/SRR2584863_1.trim.sub.fastq to /tmp/flyte-4jp7wtbm/sandbox/local_flytekit/3b8d7234fe9c7d3e37ec78af9c592208/SRR2584863_1.trim.sub.fastq (recursive=False).
+
+Original exception: failed to get presigned url for `latch:///wgs/trimmed_fastqs/SRR2584863_1.trim.sub.fastq`
+====================
+```
+
+* The error tells us that there is no file called `/wgs/trimmed_fastqs/SRR2584863_1.trim.sub.fastq` on Latch. Referencing the actual file paths of the trimmed FastQs, we can see that the filepath is indeeed wrong, with the correct filepath being `/wgs/data/SRR2584863_1.trim.sub.fastq`. 
```

Comment:
```suggestion
This error tells us that there is no file called `/wgs/trimmed_fastqs/SRR2584863_1.trim.sub.fastq` in the Latch Console. Referencing the actual file paths of the trimmed FastQs, we can see that their paths are indeed wrong, with the correct paths being `/wgs/data/SRR2584863_1.trim.sub.fastq`, etc (listed under `/wgs/data` instead of `/wgs/trimmed_fastqs`). 
```